src/pages/resources/migration/android/migrate-to-3x.md

```diff
@@ -168,3 +185,9 @@ MobileCore.trackState("view name", mapOf("&&events" to "event5,event2=2"))
   }
 }
 ```
+
+## Frequently asked questions
+
+### Why do I see 'unresolved reference' errors related to `MessagingDelegate` when upgrading to 3.x SDK?
+
+The Mobile Core 3.x SDK for Android includes changes to SDK presentation management that break compatiblity with earlier versions of the SDK. `com.adobe.marketing.mobile.services.MessagingDelegate` and its usage has been removed in favor of `com.adobe.marketing.mobile.services.ui.PresentationDelegate`. If your application uses `MessagingDelegate` for granular control of in-app messages, refer to this [page](../../../edge/adobe-journey-optimizer/in-app-message/tutorials/messaging-delegate/) for more details on using `PresentationDelegate`.
```

Comment:
Will need to change the hyperlink if AJO docs are updated to a different path.
cc @spoorthipujariadobe

Reply:
Since we're keeping both `MessagingDelegate` and `PresentationDelegate` implementation tutorials, the page name can remain the same. I will check with Steve and confirm